### PR TITLE
Docs changes

### PR DESCRIPTION
**Pull request by kaedroho**

### CODE REVIEW

File: docs/howto/settings.rst [hunk 1]
```diff
@@ -89,9 +87,6 @@ Wagtail requires several Django app modules, third-party apps, and defines sever
 Third-Party Apps
 ----------------
 
-``south``
-  Used for database migrations. See `South Documentation`_.
-
 .. _South Documentation: http://south.readthedocs.org/en/latest/
```
Comment (zerolab) [inline docs/howto/settings.rst]: Should this not be removed as well?

Comment (kaedroho) [inline docs/howto/settings.rst]: Oops, will fix. Thanks!

File: docs/howto/settings.rst [hunk 2]
```diff
@@ -165,10 +155,6 @@ Search
 
 .. code-block:: python
 
-  # Override the search results template for wagtailsearch
-  WAGTAILSEARCH_RESULTS_TEMPLATE = 'myapp/search_results.html'
-  WAGTAILSEARCH_RESULTS_TEMPLATE_AJAX = 'myapp/includes/search_listing.html'
```
Comment (zerolab) [inline docs/howto/settings.rst]: These are still used. Why not leave a reference in the code example?

Comment (kaedroho) [inline docs/howto/settings.rst]: This config should be done in urls.py now. I'll add an example there

Comment (kaedroho) [inline docs/howto/settings.rst]: Need to update this here too: http://docs.wagtail.io/en/latest/core_components/search/searching.html

Comment (kaedroho) [inline docs/howto/settings.rst]: In fact, I'll move the search stuff into a separate PR. Theres a few other things need updating there...

**Pull request merged**